### PR TITLE
849559 Prepare ug content for annotation support .net ma UI cartesian chart

### PR DESCRIPTION
**Pull request by NitheeshkumarThangaraj**
### Description ###

Provided the user guide for annotation support in .Net Maui.

- Provided the code snippet and content for below annotation
  * Line
  * Vertical
  * Horizontal
  * Ellipse
  * Rectangle
  * Text
  * View

### Output ###

Screenshot 2024-06-03 082225

Screenshot 2024-06-03 075315

Screenshot 2024-06-03 075408

Screenshot 2024-06-03 075425

Screenshot 2024-06-03 075442

Screenshot 2024-06-03 075520

Screenshot 2024-06-03 075546

Screenshot 2024-06-03 075654

Screenshot 2024-06-03 075810

Screenshot 2024-06-03 080102

Screenshot 2024-06-03 080611

### CODE REVIEW

**Review by SaiyathAliFathima — CHANGES_REQUESTED** [review]
@NitheeshkumarThangaraj  , remove the legend icon from all demo image

**Comment by NitheeshkumarThangaraj** [comment]
> @NitheeshkumarThangaraj , remove the legend icon from all demo image

@SaiyathAliFathima , okay

File: MAUI/Cartesian-Charts/Annotation.md [hunk 1]
```diff
@@ -0,0 +1,743 @@
...
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Annotation in MAUI Chart](Annotation_image/ellipse_annotation.png)
+
+## Positioning the annotation
+
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:64]: Improve the content - https://help.syncfusion.com/wpf/charts/annotations#positioning-the-annotation  , https://help.syncfusion.com/flutter/cartesian-charts/annotations#positioning-the-annotation

File: MAUI/Cartesian-Charts/Annotation.md [hunk 2]
```diff
@@ -0,0 +1,743 @@
...
+[Annotations]() can be positioned in the plot area based on the [X1]() and [Y1]() properties. For shape annotations, specify the [X2]() and [Y2]() properties, if needed. The X and Y values can be specified using axis units or pixel units, which can be identified by the [CoordinateUnit]() property.
+
+## Positioning based on CoordinateUnit as axis
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:67]: Why do we need to mention separate categories for pixel and axis range base? Update the code snippet as we discussed.

File: MAUI/Cartesian-Charts/Annotation.md [hunk 3]
```diff
@@ -0,0 +1,743 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Annotation in pixel value in MAUI Chart](Annotation_image/rectangle_pixel_value.png)
+
+## Adding annotation for multiple axes
+
+When there are multiple axes, annotations can also be added to a particular axis by using the [XAxisName]() and [YAxisName]() properties. This can be shown in the code snippet below:
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:158]: image


ensure the grammar

File: MAUI/Cartesian-Charts/Annotation.md [hunk 4]
```diff
@@ -0,0 +1,743 @@
...
+        chart.Annotations.Add(text);
+
+        this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Text annotation in MAUI Chart](Annotation_image/text_annotation.png)
+## Text Customization
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:255]: @NitheeshkumarThangaraj  ,I have provided the suggestion to use sentences as MAUI customization categories, like plot band and axis level, which we already discussed during the scrum call. Why is this not addressed?

File: MAUI/Cartesian-Charts/Annotation.md [hunk 5]
```diff
@@ -0,0 +1,743 @@
...
+
+## Rectangle annotation
+
+The [RectangleAnnotation]() is used to draw a rectangle or a square at specific points in the chart area.
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+        <chart:SfCartesianChart>
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:298]: align the code snippet

File: MAUI/Cartesian-Charts/Annotation.md [hunk 6]
```diff
@@ -0,0 +1,743 @@
...
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Ellipse annotation in MAUI Chart](Annotation_image/default_ellipse.png)
+
+N> When the [X2]() and [Y2]() properties of the [EllipseAnnotation]() are set, the [Height]() and [Width]() properties do not work.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:369]: improve the note sentence

File: MAUI/Cartesian-Charts/Annotation.md [hunk 7]
```diff
@@ -0,0 +1,743 @@
...
+
+## Line annotation
+
+The [LineAnnotation]() is used to draw a line at specific points in the chart area.
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+        <chart:SfCartesianChart>
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:379]: align the code snippet

File: MAUI/Cartesian-Charts/Annotation.md [hunk 8]
```diff
@@ -0,0 +1,743 @@
...
+    chart.Annotations.Add(line);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+   
+![Line Annotation in MAUI Chart](Annotation_image/line_annotation.png)
+## Adding arrow to line annotation
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:411]: Explain the line cap , after the horizontal and vertical line

File: MAUI/Cartesian-Charts/Annotation.md [hunk 9]
```diff
@@ -0,0 +1,743 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Ellipse annotation in MAUI Chart](Annotation_image/default_ellipse.png)
+
+N> When the [X2]() and [Y2]() properties of the [EllipseAnnotation]() are set, the [Height]() and [Width]() properties do not work.
+
+## Line annotation
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:371]: Mention note for line stroke and Fill behavior

File: MAUI/Cartesian-Charts/Annotation.md [hunk 10]
```diff
@@ -0,0 +1,743 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+![Vertical and horizontal Line Annotation in MAUI Chart](Annotation_image/horizontal_vertical_annotation.png)
+
+## Displaying axis label for vertical and horizontal line annotations
+
+The [VerticalLineAnnotation]() and [HorizontalLineAnnotation]() display the axis labels where the line is placed. This feature can be enabled by setting the [ShowAxisLabel]() property to true, as shown in the code snippet below:
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:502]: Improve the sentence , refer -https://help.syncfusion.com/maui/cartesian-charts/trackball#show-or-hide-the-series-label

File: MAUI/Cartesian-Charts/Annotation.md [hunk 11]
```diff
@@ -0,0 +1,743 @@
...
+{% endtabs %}  
+
+![Vertical and horizontal Line Annotation with axis label in MAUI Chart](Annotation_image/axis_label.png)
+
+## Customizing axis label
+
+The default appearance of the axis label can also be customized using the [AxisLabelStyle]() property. The following styles are used to customize the axis label:
+
+The [ChartLabelStyle]() property allows you to customize the axis labels by changing the font family, font size, font attributes, and text color. It is the default style for the [AxisLabelStyle]() property. The following properties can be used to customize the axis label:
+
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:551]: @NitheeshkumarThangaraj  , I have provided the suggestion to use sentences as MAUI customization categories, like plot band and axis level, which we already discussed during the scrum call. Why is this not addressed?

File: MAUI/Cartesian-Charts/Annotation.md [hunk 12]
```diff
@@ -0,0 +1,743 @@
...
+{% endhighlight %}
+
+{% endtabs %}  
+
+![Vertical and horizontal Line Annotation with linecap in MAUI Chart](Annotation_image/vertical_horizontal_linecap.png)
+
+## Adding text in shape annotation
+
+For all shape annotations, the text can be displayed using the [Text]() property.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:608]: Seems Xamarin and WPF content , improve the readability

File: MAUI/Cartesian-Charts/Annotation.md [hunk 13]
```diff
@@ -0,0 +1,743 @@
...
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Annotation in MAUI Chart](Annotation_image/ellipse_annotation.png)
+
+## Customizing text in shape annotation
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:649]: Updated the property description -https://help.syncfusion.com/maui/cartesian-charts/axis/axislabels#label-customization

File: MAUI/Cartesian-Charts/Annotation.md [hunk 14]
```diff
@@ -0,0 +1,743 @@
...
+* [Font]() – used to change the text size, font family, and font weight. (This is deprecated API. Use FontSize, FontFamily, and FontAttributes properties instead of this.)
+* [FontFamily]() - used to change the font family for the text.
+* [FontAttributes]() - used to change the font style for the text.
+* [FontSize]() - used to change the font size for the text.
+* [HorizontalTextAlignment]() - Used to align the text horizontally.
+* [VerticalTextAlignment]() - Used to align the text vertically.
+
+## View annotation
+
+The [ViewAnnotation]() allows you to add annotations in the form of your own custom view using the [View]() property at a specific area of interest in the chart area. The [ViewAnnotation]() can also be aligned using the [VerticalAlignment]() and [HorizontalAlignment]() properties.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:665]: The ViewAnnotationallows you to add annotations in the form of your own custom view using the [View]() property at a specific area of interest in the chart area. Additionally, the ViewAnnotation can be aligned using the VerticalAlignment and HorizontalAlignment properties.

**Review by SaiyathAliFathima — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Annotation.md (hunk 14 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md]: @SaiyathAliFathima , changed

File: MAUI/Cartesian-Charts/Annotation.md (hunk 13 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:607]: @SaiyathAliFathima , updated as like in maui

File: MAUI/Cartesian-Charts/Annotation.md (hunk 11 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:510]: @SaiyathAliFathima , addressed and updated the content

File: MAUI/Cartesian-Charts/Annotation.md (hunk 2 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md]: @SaiyathAliFathima , removed the axis based code and content

File: MAUI/Cartesian-Charts/Annotation.md (hunk 4 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:212]: @SaiyathAliFathima , addressed the content as per in maui

File: MAUI/Cartesian-Charts/Annotation.md (hunk 5 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md]: @SaiyathAliFathima , aligned the code snippet

File: MAUI/Cartesian-Charts/Annotation.md (hunk 7 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md]: aligned

File: MAUI/Cartesian-Charts/Annotation.md [hunk 15]
```diff
@@ -0,0 +1,743 @@
...
+
+The following annotations are supported in [SfCartesianChart]():
+
+* Text annotation
+* Shape annotation
+* View annotation
+
+## Adding Annotations
+
+You can create an instance of any type of annotation and add it to the Annotations collection. For example, the [EllipseAnnotation]() can be added.
```
[the remark this answers is not in the document]
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:22]: Make some changes with AI

File: MAUI/Cartesian-Charts/Annotation.md (hunk 3 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md]: Ensured the grammer

File: MAUI/Cartesian-Charts/Annotation.md (hunk 6 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md]: Changed the sentence

File: MAUI/Cartesian-Charts/Annotation.md (hunk 8 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md]: ok

File: MAUI/Cartesian-Charts/Annotation.md (hunk 9 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:329]: Mentioned the note for fill for line ,horizontal and vertical

File: MAUI/Cartesian-Charts/Annotation.md (hunk 10 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md]: Make some content changes with AI

File: MAUI/Cartesian-Charts/Annotation.md (hunk 12 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:523]: Make some content changes with AI

File: MAUI/Cartesian-Charts/Annotation.md [hunk 16]
```diff
@@ -0,0 +1,659 @@
...
+
+{% endtabs %}
+
+![Text annotation in MAUI Chart](Annotation_image/Text_annotation.png)
+
+## Text Customization
+
+The [TextAnnotation]() can be customized using the [LabelStyle]() property. The [LabelStyle]() property provides options to customize the font-family, font-size, font-attributes and text color of axis labels. The following properties are used to customize the text:
+
+* [TextColor]() - Gets or sets the color for the text of the label.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:216]: Gets or sets the text color for the label.

File: MAUI/Cartesian-Charts/Annotation.md [hunk 17]
```diff
@@ -0,0 +1,659 @@
...
+* [BackgroundColor]() -Gets or sets the background color of the labels.
+* [Margin]() - Gets or sets the margin of the label to customize the appearance of label.
+* [FontFamily]() -  Gets or sets the font family name for the label.
+* [FontAttributes]() - Gets or sets the font style for the label.
+* [FontSize]() - Gets or sets the font size for the label.
+* [HorizontalTextAlignment]() - Aligns the text horizontally at the Start, Center, or End.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:222]: use same as above

File: MAUI/Cartesian-Charts/Annotation.md [hunk 18]
```diff
@@ -0,0 +1,659 @@
...
+* [VerticalTextAlignment]() - Aligns the text vertically at the Start, Center, or End.
+
+## Shape annotation
+
+The [ShapeAnnotation]() allows you to add annotations in the form of shapes such as rectangles, ellipses, horizontal lines, vertical lines, etc., at specific areas of interest in the chart area.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:227]: add line too

File: MAUI/Cartesian-Charts/Annotation.md [hunk 19]
```diff
@@ -0,0 +1,659 @@
...
+
+* [RectangleAnnotation]() - Draws a rectangle over the chart area.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:229]: Start with It used to draw ,

File: MAUI/Cartesian-Charts/Annotation.md [hunk 20]
```diff
@@ -0,0 +1,659 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Rectangle annotation in MAUI Chart](Annotation_image/Rectangle_annotation.png)
+
+## Ellipse annotation
+
+The [EllipseAnnotation]() is used to draw an oval or a circle at specific points in the chart area. You can also specify the height and width of the [EllipseAnnotation]() by using the Height and Width properties, respectively.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:289]: add link for height and width

File: MAUI/Cartesian-Charts/Annotation.md [hunk 21]
```diff
@@ -0,0 +1,659 @@
...
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+    <chart:SfCartesianChart>
+    ...
+    <chart:SfCartesianChart.Annotations>
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:297]: add height and width in code

File: MAUI/Cartesian-Charts/Annotation.md [hunk 22]
```diff
@@ -0,0 +1,659 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+![Vertical and horizontal Line Annotation in MAUI Chart](Annotation_image/Horizontal_vertical_annotation.png)
+
+## Displaying axis label for vertical and horizontal line annotations
+
+The [VerticalLineAnnotation]() and [HorizontalLineAnnotation]() display the axis labels where the line is placed. The [ShowAxisLabel]() property is used to show or hide the axis’s label. This feature can be enabled by setting the [ShowAxisLabel]() property to true, as shown in the code snippet below:
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:416]: The VerticalLineAnnotation and HorizontalLineAnnotation allow for the display of axis labels at the line's position using the ShowAxisLabel property. The default value of the ShowAxisLabel property is "False."

File: MAUI/Cartesian-Charts/Annotation.md [hunk 23]
```diff
@@ -0,0 +1,659 @@
...
+
+{% endtabs %}  
+
+![Vertical and horizontal Line Annotation with axis label in MAUI Chart](Annotation_image/axis_label.png)
+
+## Axis label customization
+
+The default appearance of the axis label can also be customized using the [AxisLabelStyle]() property. The following styles are used to customize the axis label:
+
+The [ChartLabelStyle]() property allows you to customize the axis labels by changing the font family, font size, font attributes, and text color. It is the default style for the [AxisLabelStyle]() property. The following properties can be used to customize the axis label:
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:464]: Remove this as we discussed 
image

File: MAUI/Cartesian-Charts/Annotation.md [hunk 24]
```diff
@@ -0,0 +1,659 @@
...
+        <chart:VerticalLineAnnotation X1="2.5" LineCap="Arrow"/>
+        <chart:HorizontalLineAnnotation Y1="25" LineCap="Arrow"/>
+    </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %} 
+
+{% highlight c# %}
+
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:493]: update the code snippet as XAML

File: MAUI/Cartesian-Charts/Annotation.md [hunk 25]
```diff
@@ -0,0 +1,659 @@
...
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+![Vertical and horizontal Line Annotation with linecap in MAUI Chart](Annotation_image/Horizontal_vertical_with_cap.png)
+
+N> The [LineCap]() is applicable to line, horizontal, and vertical annotations by setting [LineCap]() to '[Arrow]()'.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:519]: improve the enum description

File: MAUI/Cartesian-Charts/Annotation.md [hunk 26]
```diff
@@ -0,0 +1,659 @@
...
+
+The [Text]() in shape annotation also can be customized by using the [LabelStyle]() property. The [LabelStyle]() property provides options to customize the font-family, font-size, font-attributes and text color of axis labels. The following properties are used to customize the text:
+
+* [TextColor]() - Gets or sets the color for the text of the label.
+* [Background]() - Gets or sets the background color of the labels.
+* [Margin]() - Gets or sets the margin of the label to customize the appearance of label.
+* [FontFamily]() - Gets or sets the font family name for the label.
+* [FontAttributes]() - Gets or sets the font style for the label.
+* [FontSize]() - Gets or sets the font size for the label.
+* [HorizontalTextAlignment]() - Used to align the text horizontally.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Annotation.md:574]: start with gets or sets

**Review by SaiyathAliFathima — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Annotation.md [hunk 27]
```diff
@@ -0,0 +1,659 @@
...
+
+{% endtabs %}
+
+![Annotation in MAUI Chart](Annotation_image/Adding_annotation.png)
+
+## Positioning the annotation
+
+[Annotations]() can be positioned in the plot area based on the [X1]() and [Y1]() properties. For shape annotations, specify the [X2]() and [Y2]() properties, if needed. The X and Y values can be specified using axis units or pixel units, which can be identified by the [CoordinateUnit]() property.
+ 
+## Positioning based on CoordinateUnit as pixels
```
[the remark this answers is not in the document]
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:67]: rem0ved

File: MAUI/Cartesian-Charts/Annotation.md (hunk 16 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:216]: changed

File: MAUI/Cartesian-Charts/Annotation.md (hunk 17 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:222]: updated

File: MAUI/Cartesian-Charts/Annotation.md (hunk 18 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:227]: added the line

File: MAUI/Cartesian-Charts/Annotation.md (hunk 19 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:229]: Updated with used to

File: MAUI/Cartesian-Charts/Annotation.md (hunk 20 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:289]: added link for width and height

File: MAUI/Cartesian-Charts/Annotation.md (hunk 21 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:297]: Added the code

File: MAUI/Cartesian-Charts/Annotation.md (hunk 22 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:416]: Updated the content

File: MAUI/Cartesian-Charts/Annotation.md (hunk 23 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:464]: removed the content

File: MAUI/Cartesian-Charts/Annotation.md (hunk 25 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/Annotation.md:519]: removed by setting content

**Review by SaiyathAliFathima — APPROVED** [review]

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]
PR not provided to development, Annotation.md not included to toc file.

**Pull request closed without merging**